hack/mock-based-tests.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+openshift admin new-project mock --description="This is an example project to demonstrate OpenShift v3" --admin="e2e-user"
```

Comment:
When I first start to write mock-based-tests.sh was just a demo to show how the origin/mock-app work and this script is called from within test-end-to-end.sh. So mock-based-tests.sh is not self-contained, but I can change it.